### DISCUSSION TITLE
How many load cells to buy?

### DISCUSSION THREAD

**Discussion by sozonnyk** [Zer0-bit/gaggiuino#201] [Q&A] [answered]
Hi @Zer0-bit,
It is not clear how many load cells you are planning to use. I assume you want to have them for both water tank and the cup?
Could you please clarify it, so I can buy it in advance?

By the way, big thank you for creating and maintaining the project!

**Comment by shigadeyo** [comment]
@sozonnyk ... Refer to the Wiki

**Reply by sozonnyk** [reply]
Quoting Wiki: 

> [HX711 AND 500g Load Cells](https://www.aliexpress.com/item/32670225988.html?spm=a2g0o.9042311.0.0.e1ec4c4deG81E4) - TWO OF EACH

So 2x 500g ?
Or how else to understand "HX711 and 500g" and "two of each" ?

**Comment by shigadeyo** [comment, marked as answer]
Two HX711 controllers and two 500g load cells are needed.

**Reply by sozonnyk** [reply]
Thank you. For some reason I decided that the load cell and a controller are bundled together.
